make/langtools/netbeans/nb-javac/nbproject/project.properties

```diff
@@ -1,7 +1,7 @@
-jdk.git.url=https://github.com/openjdk/jdk22u
-jdk.git.commit=jdk-22.0.1+8
-nb-javac-ver=${jdk.git.commit}
-#nb-javac-ver=jdk-21u
+jdk.git.url=https://github.com/openjdk/jdk
+jdk.git.commit=jdk-23+26
+#nb-javac-ver=${jdk.git.commit}
+nb-javac-ver=dev
```

Comment:
Dušane, why do you set the nb-javac-ver to dev? Does not it [produce strange results](https://github.com/JaroslavTulach/nb-javac/actions/runs/9480584618/job/26121592268?pr=23#step:7:224) then? Do we really want to publish `nb-javac-dev-impl.jar` on Maven central?

Reply:
You are right. Fixed.

Reply:
Looks [good](https://github.com/JaroslavTulach/nb-javac/actions/runs/9495124378/job/26166753760?pr=23#step:7:223).